.gitignore

```diff
@@ -27,5 +27,5 @@ build/
 /src/main/resources/LICENSE
 /src/main/resources/NOTICE
 # Generated
-/common/src/main/resources/common.properties
+/src/main/resources/project.properties
```

Comment:
Don't need `common.properties` - because we coalesced the projects into one project. The `project.properties` is generated and so we don't need to track is as source.